flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -695,6 +695,7 @@ void restoreInternal() throws Exception {
         }
         isRestoring = true;
         closedOperators = false;
+        getEnvironment().getMetricGroup().getIOMetricGroup().markTaskInitialize();
```

Comment:
This marks the **start** of the initialization, right?
Should we call it something like `markTaskInitializationStarted`? (I was confused at first)

Reply:
Yes, `markTaskInitializationStarted ` is better 👍